Dockerfile

```diff
@@ -70,6 +70,11 @@ RUN git clone https://github.com/openmm/openmm.git --branch "${OPENMM_VERSION}"
     make -j "$(nproc)" install && \
     make PythonInstall
 
+# Install pre-commit and cache hooks
+RUN pip install --no-cache-dir pre-commit
+COPY .pre-commit-config.yaml /code/timemachine/
+RUN cd /code/timemachine && git init . && pre-commit install-hooks
```

Comment:
Do we want to use `git init --bare .` here, since we won't actually be using the working tree?

Reply:
Makes sense, changed in 4cee42a

Reply:
Hmm, seems like pre-commit doesn't like working with a bare repo (see [log](https://gitlab.com/relaytx/physics-platform/timemachine/-/pipelines/484885708)). I reverted the change in 23e3a49.

(note this is just a hack to make pre-commit happy; it shouldn't need the git repo for anything when installing hooks, but apparently does some startup checks that require the presence of a git repo)